iati/core/tests/test_resources.py

```diff
@@ -43,49 +43,21 @@ def test_find_schema_paths(self):
             assert path[-4:] == iati.core.resources.FILE_SCHEMA_EXTENSION
             assert iati.core.resources.PATH_SCHEMAS_202 in path
 
-    @pytest.mark.parametrize('name,location', [
-        ('Name', None),
-        ('Name', 'embedded'),
-        ('Name', 'non-embedded'),
-        ('Name.xml', None),
-        ('Name.xml', 'embedded'),
-        ('Name.xml', 'non-embedded'),
+    @pytest.mark.parametrize('name', [
```

Comment:
Now that `location` has been removed, duplicates could be removed from this list.